Benchmarking Notes

```diff
@@ -0,0 +1,126 @@
+| 10000999 | velit-aut-similique 999 |           999 | http://lorempixel.com/640/480/sports |       486 |       455 |             1 |       1 |
++----------+-------------------------+---------------+--------------------------------------+-----------+-----------+---------------+---------+
+1 row in set (17.80 sec)
+
+//query by name from late entry
+mysql> select * from chairs where name = "totam-ipsum-unde 9999999";
++----------+--------------------------+---------------+------------------------------------+-----------+-----------+---------------+---------+
+| id       | name                     | collection_id | url                                | max_price | min_price | limited_offer | on_sale |
++----------+--------------------------+---------------+------------------------------------+-----------+-----------+---------------+---------+
+| 49999999 | totam-ipsum-unde 9999999 |       9999999 | http://lorempixel.com/640/480/city |       998 |       246 |             1 |       1 |
++----------+--------------------------+---------------+------------------------------------+-----------+-----------+---------------+---------+
+1 row in set (17.95 sec)
+
+//july 25 measure csv generation with npm script with 16GB memmory
+node --max-old-space-size=16384 database/csvgen1.js & node --max-old-space-size=16384 database/csvgen2.js & node --max-old-space-size=16384 database/csvgen3.js & node --max-old-space-size=16384 database/csvgen4.js & node --max-old-space-size=16384 database/csvgen5.js,
+in parallel makes 5 10M large csv in 8m33s
+
+node --max-old-space-size=16384 database/csvgen1.js; node --max-old-space-size=16384 database/csvgen2.js; node --max-old-space-size=16384 database/csvgen3.js; node --max-old-space-size=16384 database/csvgen4.js; node --max-old-space-size=16384 database/csvgen5.js",
+in sequence makes 5 10M large csv in 8m45s
+
+seeds database in 3 min 41.66 sec
+
+
+//July 26
+add more uniqueness to name field;
+look into adding index for faster queries
+used $
+alter table chairs add index collection_id (collection_id);
+
+combined generation of csv, cat csv, insert csv, index db into one npm script
+  generates, inserts, and indexes db in 14min26seconds with minimal ram usage
+
+testing queries after indexing
+  sequelize findAll where collection_id=...
+    Executing (default): SELECT `id`, `name`, `collection_id`, `url`, `max_price`, `min_price`, `limited_offer`, `on_sale` FROM `chairs` AS `chairs` WHERE `chairs`.`collection_id` = 9999999;
+    GET /products-id/9999999 200 4.600 ms - 910
+  sequelize raw query by collection_id=...
+    Executing (default): SELECT * FROM chairs where collection_id=9999997
+    GET /products-idraw/9999997 200 1.894 ms - 918
+  sequelize raw query by id= or id= or id= or id= or id=
+    Executing (default): SELECT * FROM chairs where id=9999998 or id=19999998 or id=29999998 or id=39999998 or id=49999998
+    GET /products-raw/9999998 200 2.644 ms - 919
+
```

Comment:
Nice, I like that you have all these notes and benchmarks recorded and in the repo.